DisCatSharp/Net/Rest/DiscordApiClient.cs

```diff
@@ -2044,7 +2044,7 @@ internal async Task<DiscordMessage> EditMessageAsync(ulong channel_id, ulong mes
                     attachments_new.Add(att);
                     file_id++;
                 }
-                if (attachments.HasValue && attachments.Value?.Count > 0)
+                if (attachments.HasValue && attachments.Value?.Count() > 0)
```

Comment:
wouldnt `attachments.Value?.Any()` be better?
asking for a friend

Reply:
You're right! Thanks.
Fixing when I work on that again.